### DISCUSSION TITLE
Who owns the TravisCI GITHUB_TOKEN over at theia-ide/theia-apps?

### DISCUSSION THREAD

**Discussion by thegecko** [eclipse-theia/theia#7967] [General] [answered]
Can one of the maintainers confirm with me who owns the GITHUB_TOKEN used in the [TravisCI](https://travis-ci.org/theia-ide/theia-apps) build for [theia-ide/theia-apps](https://github.com/theia-ide/theia-apps)?

Thanks!

@akosyakov @marcdumais-work @vince-fugnitto @kittaakos

**Comment by marcdumais-work** [comment, marked as answer]
Hi Rob,

I think it might be mine.

**Reply by thegecko** [reply]
Thanks,  I'll drop you a line :)
